SpinTimer.cpp

```diff
@@ -46,7 +48,8 @@ SpinTimer::SpinTimer(SpinTimerAction* action, bool isRecurring, unsigned long ti
 {
   SpinTimerContext::instance()->attach(this);
 
-  if (0 < m_delayMillis)
+  if( (isAutostart == SpinTimer::IS_AUTOSTART) && 
```

Comment:
The comparison with `SpinTimer::IS_AUTOSTART`, which is a bool, could be omitted - just `if (isAutostart)` would be enough.